### PR TITLE
Track when learner roster page changes.

### PR DESCRIPTION
**Pull request by dsjen**
This adds a tracking event for when the user changes page on the roster view.

@ajpal, please review.

### CODE REVIEW

**Comment by ajpal** [comment]
👍

**Comment by dsjen** [comment]
Please review, @AlasdairSwan.

CC: @ajpal @thallada

File: analytics_dashboard/static/apps/learners/roster/views/paging-footer.js [hunk 1]
```diff
@@ -26,8 +26,24 @@ define(function (require) {
             forward: {title: 'Next', label: '<i class="fa fa-step-forward" aria-hidden="true"></i>'},
             fastForward: {title: 'Last', label: '<i class="fa fa-fast-forward" aria-hidden="true"></i>'}
         },
+        initialize: function (options) {
+            Backgrid.Extension.Paginator.prototype.initialize.call(this, options);
+            this.options = options || {};
+        },
+        render: function () {
+            var self = this;
```
Comment (AlasdairSwan) [inline analytics_dashboard/static/apps/learners/roster/views/paging-footer.js]: We should not need to alias `this`, instead you can either bind `this` to the each loop or alias `this.options.trackingModel`.

Comment (dsjen) [inline analytics_dashboard/static/apps/learners/roster/views/paging-footer.js]: Right on!

**Comment by dsjen** [comment]
@AlasdairSwan -- I updated the PR to remove the reference to `self`.  Please let me know if there's anything else.

**Comment by AlasdairSwan** [comment]
Looks good @dsjen  👍

**Pull request merged**